photo-points/Services/AdminReviewServices.cs

```diff
@@ -0,0 +1,35 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using photo_points.Models;
+
+namespace photo_points.Services
+{
+    // You may need to install the Microsoft.AspNetCore.Http.Abstractions package into your project
+    public class AdminReviewServices : IAdminReviewServices
+    {
+
+        public bool approve(long captureID, byte photo, DateTime captureDate, bool Approved)
```

Comment:
Are you passing a the true parameter from the Approved?

Reply:
yes